draft-ietf-tls-tls13.md

```diff
@@ -1415,7 +1413,7 @@ one extension (at least "key_share" or "pre_shared_key").
 max_supported_version
 : The latest (highest valued) version of the TLS protocol offered by the
   client. This SHOULD be the same as the latest version supported. For this
-  version of the specification, the version will be { 3, 4 }. (See
+  version of the specification, the version ID will be { 3, 4 }. (See
```

Comment:
"the version will be 0x0304 (TLS 1.3)"

No point mucking around with the struct.